website/docs/cloud-docs/overview/migrate-teams-standard.mdx

```diff
@@ -0,0 +1,46 @@
+---
+page_title: Migrate from legacy Team Edition to new Standard Edition - Plans and Features - Terraform Cloud
+description: >-
+  Migrate your Terraform Cloud from the legacy Teams Edition to the 
+    new Standard Edition.
+tfc_only: true
+---
+
+# Migrate from legacy Team Edition to new Standard Edition
+
+To migrate from the legacy Terraform Cloud **Team** Edition to the new **Standard** Edition, you must create or log in to your HashiCorp Cloud Platform account. If you already have a Terraform Cloud account, you must instead [link](/terraform/cloud-docs/users-teams-organizations/users#linked-hcp-and-terraform-cloud-accounts) it with your [HashiCorp Cloud Platform billing account](/hcp/docs/hcp/admin/billing). 
+
+First, navigate to your organization's **Settings** page, then click **Plan & billing**.
+
+![Screenshot: Notification reading that the user must link their HashiCorp Cloud Platform account to change their plan](/img/docs/tfc-pnb-link-hcp.png)
+
+Next, click **Edit plan**.
+
+![Screenshot: A highlight of the "Edit Plan" button on the plan summary screen](/img/docs/tfc-edit-plan.png)
+
+## Create or select a HashiCorp Cloud Platform organization
+
+If your HashiCorp Cloud Platform account is new or recently created, you can automatically create a new HCP organization. All HCP Organizations include $50 free Trial credits.  
+
+![Screenshot: A screenshot prompting the user to create an HCP organization](/img/docs/tfc-create-hcp-org.png)
+
+## Select Standard plan
+
+The pay-as-you-go offering only includes the **Standard** Edition feature set.  To migrate to the **Plus** Edition, contact [HashiCorp Sales](https://www.hashicorp.com/contact-sales).
+
+Select the **Standard** plan, then click **Next**.
+
+![Screenshot: The "Select Plan" screen showing the three Terraform Cloud plans: Free, Standard, and Plus. The Standard option is highlighted and includes an estimated cost.](/img/docs/tfc-edit-plan-standard.png)
+
+## Plan Summary
+
+The plan summary screen shows a cost estimate and provides a link to add a credit card. No credit card is required to try Terraform Cloud Standard Edition.
+The new **Standard** Edition feature set is available once you activate the new plan. Usage and consumption will start immediately. Refer to [Usage Metering and Cost Calculation](/hcp/docs/hcp/admin/billing/flex-billing#usage-metering-and-cost-calculation) for more information on how to review your organization's usage and consumption.
+
+## Viewing Usage, Consumption, & Billing Invoices
+
+Billing is managed on HashiCorp Cloud Platform. There you can:
+
+- [View usage](/hcp/docs/hcp/admin/billing#hcp-usage) related to managed resources
+- Update [credit card payment](/hcp/docs/hcp/admin/billing/payment)
+- Setup a [Flex Billing Contract](/hcp/docs/hcp/admin/billing/payment#set-up-an-entitlement-or-hashicorp-flextm-billing-contract)
```

Comment:
```suggestion
# Migrate from legacy team edition to standard edition

This topic describes how to migrate Terraform Cloud from the legacy team edition to the standard edition. Refer to [Terraform Cloud Plans and Features](/terraform/cloud-docs/overview) for an overview of the editions you can subscribe to. 

## Prerequisistes

Create or log in to your HashiCorp Cloud Platform account. If you already have a Terraform Cloud account, you must instead link it to your [HashiCorp Cloud Platform billing account](/hcp/docs/hcp/admin/billing). Refer to [Linked HCP and Terraform Cloud Accounts](/terraform/cloud-docs/users-teams-organizations/users#linked-hcp-and-terraform-cloud-accounts) for instructions.  

## Migrate editions

Complete the following steps to migrate from the team edition to the standard edition:

1. Navigate to your organization's **Settings** page, then click **Plan & billing**.
    ![Screenshot: Notification reading that the user must link their HashiCorp Cloud Platform account to change their plan](/img/docs/tfc-pnb-link-hcp.png)

1. Click **Edit plan**.

    ![Screenshot: A highlight of the "Edit Plan" button on the plan summary screen](/img/docs/tfc-edit-plan.png)

1. HCP prompts you to create a new organization if your HashiCorp Cloud Platform account is new or recently created. Click **Create HCP organization** to create a new HCP organization. All HCP Organizations include $50 free Trial credits.  

    ![Screenshot: A screenshot prompting the user to create an HCP organization](/img/docs/tfc-create-hcp-org.png)

1. When prompted, choose the **Standard** plan and click **Next**. The pay-as-you-go offering only includes the **Standard** edition feature set. To migrate to the **Plus** edition, contact [HashiCorp Sales](https://www.hashicorp.com/contact-sales).

    ![Screenshot: The "Select Plan" screen showing the three Terraform Cloud plans: Free, Standard, and Plus. The Standard option is highlighted and includes an estimated cost.](/img/docs/tfc-edit-plan-standard.png)

## Plan summary

The plan summary screen shows a cost estimate and provides a link to add a credit card. No credit card is required to try Terraform Cloud standard edition.

The **Standard** edition feature set is available once you activate the new plan. Usage and consumption start immediately. Refer to [Usage Metering and Cost Calculation](/hcp/docs/hcp/admin/billing/flex-billing#usage-metering-and-cost-calculation) for more information on how to review your organization's usage and consumption.

## View account information

You can track usage, consumption, and billing invoices on HashiCorp Cloud Platform. Refer to the following topics for additional information:

- [View usage](/hcp/docs/hcp/admin/billing#hcp-usage) related to managed resources
- Update [credit card payment](/hcp/docs/hcp/admin/billing/payment)
- Setup a [flex billing contract](/hcp/docs/hcp/admin/billing/payment#set-up-an-entitlement-or-hashicorp-flextm-billing-contract)
```
Sorry for the wholesale rewrite. The usage content type is just so ingrained. 
Is there a final step to complete the process?The instructions just kind of end. It's not always clear when the screens progress to the next step in the UI, either. For example, does the prompt in the step 3 always appear or just when you have a new organization? Are you otherwise taken to the screen to choose your edition? 
I changed the spelling to lowercase for a lot of words that don't seem like proper nouns. I'm new to TF, so there may be words that we have agreed to capitalize.